3p/ampcontext.js

```diff
@@ -308,6 +311,10 @@ export class AbstractAmpContext {
     this.tagName = context.tagName;
 
     this.embedType_ = dataObject.type || null;
+
+    if (context.initialConsentValue) {
```

Comment:
Do we need a check here? Wouldn't this just return undefined anyways?

Reply:
Ah, I think you're right. I was hoping to not set the initialConsentValue until we turn on experiment. But since we always pass `null` now, and it's not documented we should be good.